### PR TITLE
[Back to top] More relevant description

### PR DESCRIPTION
**Pull request by photodow**
**Changed**

- Back to top description

### CODE REVIEW

File: src/pages/components/back-to-top.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Back to top
-description: asdf.
+description: The Back to top component is a fixed button that allows the user to scroll to the top of a long page.
```
Comment (janchild) [inline src/pages/components/back-to-top.mdx:3]:
```suggestion
description: The Back to top component is a fixed button that allows the user to quickly navigate to the top of a page.
```

**Review by janchild — COMMENTED** [review]
@photodow just tweaked a little :)

File: src/pages/components/back-to-top.mdx (hunk 1 above)
Comment (photodow) [reply src/pages/components/back-to-top.mdx:3]: I was hoping you would tweak it. :wink: thanks!

**Review by janchild — APPROVED** [review]
:tada:

**Pull request merged**